### ISSUE TITLE
How to stop exporting?

### ISSUE THREAD

**Issue by jcubic** [hniksic/emacs-htmlize#20]
I have lot of errors on big file and keep showing then in message buffer and <kbd>M</kbd>-<kbd>g</kbd> doesn't do anything. How can I stop processing?

**Comment by hniksic** [comment]
Have you tried <kbd>C-g</kbd> instead of <kbd>M-g</kbd>?

**Comment by jcubic** [comment]
Yes I was using <kbd>C-g</kbd> it was a typo. It seems that the library is doing its processing in background so there is nothing to kill with <kbd>C-g</kbd>.

**Comment by hniksic** [comment]
Which library are you referring to? `htmlize` is pretty straightforward in that sense, it does no background work as far as I'm aware.

**Comment by jcubic** [comment]
I've downloaded htmlize.el from this repo and it didn't freezed when it processing and keep showing me tens of thousands of errors in message buffer and I need to kill emacs.
